packages/zoltan2/test/correctness/zoltanCompare.cpp

```diff
@@ -119,6 +120,15 @@ typedef Zoltan2::XpetraMultiVectorAdapter<tMVector_t> vectorAdapter_t;
 typedef Zoltan2::XpetraCrsMatrixAdapter<tMatrix_t,tMVector_t> matrixAdapter_t;
 typedef Zoltan2::EvaluatePartition<matrixAdapter_t> quality_t;
 
+// Number of ZOLTAN_ID in a zgno_t (for NUM_GID_ENTRIES)
+static constexpr int nGidEnt = sizeof(zgno_t) / sizeof(ZOLTAN_ID_TYPE);
+#define SET_ZID(n,a,b)                                            \
```

Comment:
For the future, best practice would be to capture the values of `n`, `a`, and `b` above the loop using `auto`:
```c++
const auto zoltan_zid_loop_n = n;
auto zoltan_zid_loop_a = a;
auto zoltan_zid_loop_b = b;
for (ZOLTAN_ID_LOOP = 0; ZOLTAN_ID_LOOP < n; ++ZOLTAN_ID_LOOP) {
  zoltan_zid_loop_a[ZOLTAN_ID_LOOP] = zoltan_zid_loop_b[ZOLTAN_ID_LOOP];
}
```